packages/compass/.prettierignore

```diff
@@ -3,3 +3,6 @@ dist
 coverage
 build
 src/deps
+
+# Generated
+THIRD-PARTY-NOTICES.md
```

Comment:
This got copied on my system while building the app and kept the `check` from passing locally.